src/test/java/HomeTests.java

```diff
@@ -0,0 +1,52 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class HomeTests extends BaseTest {
+
+    String createdPlaylist = "Created playlist \"vivo.\"";
+    String updatedPlaylistMsg = "Updated playlist \"VIVO.\"";
+    String deletedPlaylistMsg = "Deleted playlist \"VIVO.\"";
+
+
+    @Test(priority = 0)
+    public void createPlaylist() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+
+        loginPage.logIn();
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.